File: Module.Frontend.TPM/Client/model/tpm/retailtype/HistoricalRetailType.js
Ext.define('App.model.tpm.retailtype.HistoricalRetailType', {
    extend: 'Ext.data.Model',
    idProperty: '_Id',
    breezeEntityType: 'RetailType',
    fields: [
        { name: '_Id', type: 'string', hidden: true },
        { name: '_ObjectId', hidden: true },
        { name: '_User', type: 'string', isDefault: true },
        { name: '_Role', type: 'string', isDefault: true },
        { name: '_EditDate', type: 'date', isDefault: true },
        { name: '_Operation', type: 'string', isDefault: true },
        { name: 'Name', type: 'string', hidden: false, isDefault: true }
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'HistoricalRetailTypes',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        }
    }
});

File: Module.Persist.TPM/Model/History/HistoricalRetailType.cs
using Core.History;
using Module.Persist.TPM.Model.TPM;
using System;

namespace Module.Persist.TPM.Model.History
{
    [AssociatedWith(typeof(RetailType))]
    public class HistoricalRetailType : BaseHistoricalEntity<System.Guid>
    {
        public string Name { get; set; }

    }
}
